src/components/Game/Timer/Timer.js

```diff
@@ -2,58 +2,58 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
 import classes from './Timer.scss';
-import { computeMinutes, computeSeconds, printTwoDigits } from '../../../helpers';
+import { formatTimeLeft } from '../../../helpers';
+import { DURATION, INTERVAL } from '../../../constants/constants';
+
 
 class Timer extends Component {
     state = {
-        // 60 sec. only for testing reasons. it's gonna be more :)
-        seconds: 60,
+        timeLeft: DURATION,
     };
 
+    start = new Date().getTime();
+
     componentDidMount() {
+        this.setState({
+            timeLeft: this.getTimeLeft(new Date()),
+        });
         this.startTimer();
     }
 
     componentWillUnmount() {
         this.stopTimer();
     }
 
+    getTimeLeft(now) {
```

Comment:
`getTimeLeft` function is called twice and every time it gets `new Date()` as a first argument
so it makes me think you may want to remove the parameter and replace `now` inside the function body with `new Date()`

Reply:
of course! 🤦🏻‍♀️